utils/virtual_amiibo_file.py

```diff
@@ -222,6 +255,43 @@ def get_personality(self):
             self.dump.data = cli.dump_to_amiitools(self.dump.data)
             return "Normal"
 
+    def is_initialized(self):
+        """Checks if the amiibo was initialized by amiibo settings
+        Returns:
+            Bool: False if it hasn't been initialized, True if it has
+        """
+        result = None
+        self.dump.data = cli.amiitools_to_dump(self.dump.data)
+        if not (self.dump.data[0x14] >> 4) & 1:
+            result = False
+        else:
+            result = True
+        self.dump.data = cli.dump_to_amiitools(self.dump.data)
+        return result
+
+    def initialize_amiibo(self, mii_path: str, name: str):
+        """_summary_
+
+        Args:
+            mii_path (str): The path to the mii to use for initialization
+            name (str): The amiibo name
+
+        Raises:
+            InvalidMiiSizeError: Raises this error if the mii isn't 96 bytes.
+        """
+        self.dump.data = cli.amiitools_to_dump(self.dump.data)
+        # Sets the amiibo name to the given name
+        self.dump.amiibo_nickname = name
+        self.dump.data = cli.dump_to_amiitools(self.dump.data)
+        # Sets bit 4 of the settings byte to 1, letting the user use the amiibo in game.
+        self.dump.data[83] = self.dump.data[83] | (1 << 4)
+        self.dump.data = cli.amiitools_to_dump(self.dump.data)
+        try:
+            self.set_mii(mii_path)
+        except InvalidMiiSizeError:
+            raise InvalidMiiSizeError
+        self.dump.data = cli.dump_to_amiitools(self.dump.data)
```

Comment:
Can move bit setting line to start or end of function to remove additional cli formatting calls